job_bundles/Using_Files_In_Jobs.md

```diff
@@ -0,0 +1,1154 @@
+
+For the purpose of demonstration, consider the following hypothetical infrastructure that is set up to
+support two separate projects. To follow along, set up a farm, fleet, and two queues using the console for
+AWS Deadline Cloud and then be sure to delete these resources when you are done:
+
+1. [Farm](https://docs.aws.amazon.com/deadline-cloud/latest/userguide/farms.html#create-farm):
+    1. Named: `AssetDemoFarm`
+    2. All other settings default.
+2. Two [queues](https://docs.aws.amazon.com/deadline-cloud/latest/userguide/create-queue.html):
+    1. The first is for jobs for only the first of the two projects:
+        1. Named: `Q1`.
+        2. Job attachments: Create a new Amazon S3 bucket.
+        3. Association with customer-managed fleets: Enabled.
+        4. Run as user configuration: `jobuser` as both the user name and group name for POSIX credentials.
+        5. Queue service role: Create a new role the name `AssetDemoFarm-Q1-Role`
+        6. Default Conda queue environment: Unselect the checkbox for the default queue environment.
+        7. All other settings default.
+    2. The second is for jobs for only the second of the two projects:
+        1. Named: `Q2`.
+        2. Job attachments: Create a new Amazon S3 bucket.
+        3. Association with customer-managed fleets: Enabled.
+        4. Run as user configuration: `jobuser` as both the user name and group name for POSIX credentials.
+        5. Queue service role: Create a new role with the name `AssetDemoFarm-Q2-Role`
+        6. Default Conda queue environment: Unselect the checkbox for the default queue environment.
+        7. All other settings default.
+3. A single customer-managed [Fleet](https://docs.aws.amazon.com/deadline-cloud/latest/userguide/create-a-cmf.html)
+   that will run the jobs from both queues:
+    1. Named: `Fleet`
+    2. Fleet type: Customer-managed.
+    3. Fleet service role: Create a new role with a name of your choosing. e.g. `AssetDemoFarm-Fleet-Role`
+    4. All other settings default. Importantly, do not associate the fleet with any queues at this time.
+
+This hypothetical infrastructure has three filesystem locations that are shared between hosts via network fileshares. We
+refer to these locations by the following names:
+
+1. `FSComm` - Containing input job assets that are common to both projects.
+2. `FS1` - Containing input and output job assets for project 1.
+3. `FS3` - Containing input and output job assets for project 2.
+
+The infrastructure has three workstation configurations that we'll refer to as `WS1`, `WS2`, and `WSAll`:
+
+1. `WSAll` - A Linux-based workstation set up for developers to assist with all projects. The shared filesystem locations are:
+    1. `FSComm`: `/shared/common`
+    2. `FS1`: `/shared/projects/project1`
+    3. `FS2`: `/shared/projects/project2`
+2. `WS1` - A Windows-based workstation set up to work on only project 1. The shared filesystem locations are:
+    1. `FSComm`: `S:\`
```

Comment:
nit: recommend to use \\?\S:\ due to long file path limitations. Nested long paths here will be weird.

Reply:
Yeah, I'm not sure how well our implementation handles that. I'd have to test it. We can modify this later once we know, for certain, that would work with our implementation.